dnf-plugins-core.spec

```diff
@@ -1,6 +1,6 @@
 %{?!dnf_lowest_compatible: %global dnf_lowest_compatible 1.1.4}
 %{?!dnf_not_compatible: %global dnf_not_compatible 2.0}
-%global hawkey_version 0.6.1
+%global hawkey_version 0.7.0
```

Comment:
As you can see hawkey_version is used only in python-hawkey requirement and it will be fulfilled once the libhif 7.0 is released.